test/ClassicWebhookEventVerifierTest.php

```diff
@@ -6,33 +6,19 @@
 
 use Onfido as Onfido;
 
-class WebhookEventVerifierTest extends TestCase
+class ClassicWebhookEventVerifierTest extends TestCase
 {
     private static $rawEvent = "{\"payload\":{\"resource_type\":\"check\",\"action\":\"check.completed\",\"object\":{\"id\":\"f2302f45-227d-413d-ad61-09ec077a086a\",\"status\":\"complete\",\"completed_at_iso8601\":\"2024-04-04T09:21:21Z\",\"href\":\"https://api.onfido.com/v3.6/checks/f2302f45-227d-413d-ad61-09ec077a086a\"}}}";
     private static $webhookToken = "wU99mE6jJ7nXOLFwZ0tJymM1lpI15pZh";
 
     private static $webhookEventVerifier;
 
-    private static $expectedEvent;
-
     /**
      * Setup before running any test case
      */
     public static function setUpBeforeClass(): void
     {
         self::$webhookEventVerifier = new Onfido\WebhookEventVerifier(self::$webhookToken);
-
-        self::$expectedEvent = new Onfido\Model\WebhookEvent(
```

Comment:
Removing this since it's useless (i.e. not used in the tests).

CC: @sofia-gomes-onfido